index.html

```diff
@@ -1,42 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
- <link rel="stylesheet" href="style.css" />
-  <title>Portfolio</title>
-</head>
-<body>
-
-	<header class="navbar">
-		 <div class="mylogo">
-		<a href="#" >Elhama Tokhi</a>
-	    </div>
-		<nav class="icon">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div class="mylogo">
+        <a href="#">Elhama Tokhi</a>
+      </div>
+      <nav class="icon">
         <img src="Images/Union.png" alt="Menu-icon" />
-    	 </nav>
+      </nav>
     </header>
 
+    <main class="container">
+      <h1>
+        I'm Elhama Tokhi 
+        Glad to see 
+        you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
 
-	<main class="container">
-
-		<h1> I'm Elhama Tokhi <br/>
-        Glad to see <br/>
-        you!</h1>
-	<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-	 <h3>LET'S CONNECT</h3>
-
-		<div class="social-icons">
-		<ul>
-			<li><a href=""><img src="Images/Social/github.svg"  class="github"></a></li>
-			<li><a href=""><img src="Images/Social/Linkedin.svg" class="linkedin"></a></li>
-			<li><a href=""><img src="Images/Social/mail.svg" class="mail"></a></li>
-			<li><a href=""><img src="Images/Social/other.svg" class="other"></a></li>
-			<li><a href=""><img src="Images/Social/twitter.svg" class="twitter"></a></li>
-		</ul>
-		</div>
-
-	</main>	
-</body>
+      <div class="social-icons">
+        <ul>
+          <li>
+            <a href=""><img src="Images/Social/github.svg" class="github" alt="" /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img src="Images/Social/Linkedin.svg" class="linkedin" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href=""><img src="Images/Social/mail.svg" class="mail" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="Images/Social/other.svg" class="other" alt="" /></a>
+          </li>
+          <li>
```

Comment:
- [ ] `ul` tags usually have default inbuilt margins which is why you see this extra margin here at the start. of your social links. Kindly remove this margin so your links conform to the Figma template. 
> Yours
![image](https://user-images.githubusercontent.com/90556221/170800390-1225f858-d790-45b1-983f-d69d1969420b.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/170800425-5aee7433-e49d-45d1-a33e-007344ac8ed8.png)